src/themes/theme.json

```diff
@@ -9,46 +9,46 @@
   },
   "grid": {
     "xxs": {
-      "screensize": "<576px",
+      "screensize": "576px",
```

Comment:
"xxs" is the only breakpoint that is different from the others, it's smaller than 576px, all other values are >=